### DISCUSSION TITLE
Wine and ubuntu for orange pi 5

### DISCUSSION THREAD

**Discussion by ironiris** [Joshua-Riek/ubuntu-rockchip#122] [General]
For Rock5, Wine is provided by Armbian in the After Burner version,
I would like to install Wine on Ubuntu on Orange Pie 5, but it's not easy.
Is it possible to provide documentation (wiki) on how to install it?

**Comment by Joshua-Riek** [comment]
Sure, I have been wanting to do this for a while. I finished making a package repo to distribute kernel, firmware, and bootloader updates with apt. So I can finally move on to creating more documentation and other bug fixes. It might take me a few days to make something that works, last time I tested Wine it gave me issues.
